src/hugo/content/en-us/implementations/aws/deviceOnboarding/Provisioning-JITP/_index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Provisioning - JITP"
+weight: 10
+summary: "Command and control of a device using MQTT topics"
+---
+
+In an IoT deployment, security is often the number one concern from both device
+customers and device manufacturers. To protect and encrypt data in transit from an IoT
+device to AWS IoT Core, AWS IoT Core supports TLS-based mutual authentication
+using X.509 certificates. AWS IoT provides options to provision and onboard a large number of devices based
+on the capabilities of the device and if the devices have their unique X.509 certificate
+and private keys on them before being sold to the end customer.
+If the manufacturing chain allows the device maker to provision unique credentials into
+the device at manufacturing time, device makers can use Just in Time
+Provisioning. 
+
+To use JITP, devices should have certificates and private keys present on the device before onboarding to AWS IoT. Certificate authority (CA) is an entity that issues digital certificates. The device certificates saved on your devices must be signed with your designated CA, and that CA must be registered in AWS IoT. 
+{{% notice note %}}
+To use JITP you have to maintain your PKI infrastructure, this can also be done using managed service like Amazon Certificate Manager. The manufacturing facility should have capability to create a private key, CA signed certificate and store the key-certificate pair on the device memory.
+{{% /notice %}}
+
+## Use cases
+Just-in-time provisioning (JITP) is the recommended approach when the manufacturing chain allows the device maker to provision unique credentials into the device at manufacturing time. To implement JITP the manufacturing chain has to be trusted and should have the capability to create a device certificate signed by a self-managed CA. You can have your devices provisioned when they first attempt to connect to AWS IoT with just-in-time provisioning (JITP). 
+
+Example use cases where JITP should be used are:
+
+1. When device originates from an original device manufacturer and you have the ability to include unique key material.
+2. When maintaining public key infrastructure (PKI) is feasible.
+3. There is a trust channel between your PKI and the manufacturing facility.
+
+## Reference architecture
+
+![JITP](JITP_arch_1.png)
+The details of this flow are as follows:
+1. Private key and signed certificate pair is created using PKI. PKI can be self-managed or using a managed service like Amazon ACM. 
+2. This pair is securely copied and stored on the device memory. 
+3. Using JITP, the device connects to AWS IoT, and the certificate’s signature is verified
+against the registered CA. After verification, a provisioning template registers the Thing,
+certificate, and assigns a policy to the device. A provisioning template is a JSON document that uses parameters to describe the resources your device must use to interact with AWS IoT. 
```

Comment:
Reword _A provisioning_ template. It defines what it is but now how it's used during JITP.